system_tests/forwarder_test.go

```diff
@@ -157,7 +158,14 @@ func createSequencer(
 
 // tmpPath returns file path with specified filename from temporary directory of the test.
 func tmpPath(t *testing.T, filename string) string {
-	return filepath.Join(t.TempDir(), filename)
+	// create a unique, maximum 10 characters-long temporary directory {name} with path as $TMPDIR/{name}
+	tmpDir, _ := os.MkdirTemp("", "")
```

Comment:
Please handle error:

tmpDir, err := os.MkdirTemp()...
if err != nil {
  t.Fatalf(err)
}

Fatalf since if this errors out there's no point in continuing the test.